src/Symfony/Component/Console/Descriptor/ReStructuredTextDescriptor.php

```diff
@@ -92,7 +92,7 @@ protected function describeInputOption(InputOption $option, array $options = [])
     protected function describeInputDefinition(InputDefinition $definition, array $options = []): void
     {
         if ($showArguments = ((bool) $definition->getArguments())) {
-            $this->write("Arguments\n".str_repeat($this->subsubsectionChar, 9))."\n\n";
```

Comment:
Why this change?

Reply:
The double newline is outside the `write()` call. Seems weird to be intentional, I guess it has never done anything then